src/inputs/index.ts

```diff
@@ -1,11 +1,14 @@
 export * from "./Checkbox";
 export * from "./CheckboxGroup";
 export * from "./DateField";
+export * as MockDateField from "./DateField.mock";
 export * from "./MultiSelectField";
+export * as MockMultiSelectField from "./MultiSelectField.mock";
 export * from "./NumberField";
 export { RadioGroupField } from "./RadioGroupField";
 export type { RadioFieldOption, RadioGroupFieldProps } from "./RadioGroupField";
 export * from "./SelectField";
+export * as MockSelectField from "./SelectField.mock";
```

Comment:
Do we need these exports? Like if downstream projects are doing things like:

```
jest.mock("@homebound/beam/dist/inputs/SelectField", () => {
  return require("src/mocks/MockSelectField");
});
```

And mocking at the module level...

Granted, doesn't hurt either, just curious.

Reply:
so my thought is that once this is available we would remove all of the application's mocks and instead reference these exported mocks from beam like:
```
jest.mock("@homebound/beam/dist/inputs/SelectField", () => {
  return require("@homebound/beam/dist/inputs/SelectField.mock");
});
```
keeping all mocks centralized. This should also make it easier for us to look into providing a great abstraction (not having to define each and every mock)

Reply:
just realized I rambled on and didn't answer your question. No don't need to export them so I'll remove

Reply:
+1 to your snippet, I guess my point was that we don't need `MockSelectField` exported from an `index.ts` file then?

Some of the issues @bdow and I have been running into today has been from Beam's `index.ts` files exporting both "for production components" + "for testing utils", and so we're nudging the `index.ts` style exports to be only what's really really production code.

Reply:
funnily enough I was just looking into your comment again after following the PR and I agree to removing the mocks from being explicitly exported. I'll update